pixano/app/api/items.py

```diff
@@ -411,107 +411,118 @@ def find_top_parent_entity(entity: Entity) -> Entity | None:
                 obj["features"] = features
                 objects.append(obj)
     else:  # video
-        tracks = defaultdict(list)
-        entity_id = defaultdict(list)
+        trackid2trackletslist = defaultdict(list)  # trackid2trackletslist
```

Comment:
Can we do some refactoring so that the variable names ae more easily readable and consistent  ?
trackid2trackletslist => track_id_to_tracklet_list
for instance